sopel/irc.py

```diff
@@ -300,7 +286,7 @@ def handle_connect(self):
         self.write(('USER', self.user, '+iw', self.nick), self.name)
 
         # maintain connection
-        stderr('Connected.')
+        LOGGER.info('Connected.')
```

Comment:
This is why the `...` in the "Connecting" log entry above is appropriate. This is the end of the possibly-lengthy process started there.